index.js

```diff
@@ -63,7 +64,8 @@ Unrar.prototype.stream = function(entryname) {
   return new UnrarStream({
     entryname: entryname,
     filepath: this._filepath,
-    arguments: this._arguments
+    arguments: this._arguments,
+    $bin: this._unrarBin,
```

Comment:
why `$` in the name?

Reply:
Sorry, I am based on the implementation of node-7z for binaries, but there it is named so because it is a characteristic of this module, I will proceed to change it.

https://github.com/quentinrossetti/node-7z